src/setup-beam.js

```diff
@@ -224,10 +219,7 @@ async function getOTPVersions(osVersion) {
         .filter((x) => x.name.match(/^otp_win64_.*.exe$/))
         .forEach((x) => {
           const otpMatch = x.name.match(/^otp_win64_(.*).exe$/)
-          let otpVersion = otpMatch[1]
```

Comment:
This was moved to `getVersionFromSpec` to centralise version management/handling.